src/Goerli-DssSpell.t.base.sol

```diff
@@ -989,6 +1078,149 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
+            sourceDomain: sourceDomain,
+            targetDomain: targetDomain,
+            receiver: bytes32(uint256(uint160(address(this)))),
+            operator: bytes32(0),
+            amount: uint128(toMint),
+            nonce: 1,
+            timestamp: uint48(block.timestamp)
+        })));
+        oracleAuth.addSigners(signers);
+        oracleAuth.requestMint(TeleportGUID({
+            sourceDomain: sourceDomain,
+            targetDomain: targetDomain,
+            receiver: bytes32(uint256(uint160(address(this)))),
+            operator: bytes32(0),
+            amount: uint128(toMint),
+            nonce: 1,
+            timestamp: uint48(block.timestamp)
+        }), signatures, expectedFee, 0);
+    }
+
+    // NOTE: Only executable by forge
+    function checkTeleportFWIntegration(
+        bytes32 sourceDomain,
+        bytes32 targetDomain,
+        uint256 line,
+        address gateway,
+        address fee,
+        address escrow,
+        uint256 toMint,
+        uint256 expectedFee,
+        uint256 expectedTtl
+    ) public {
+        TeleportJoinLike join = TeleportJoinLike(addr.addr("MCD_JOIN_TELEPORT_FW_A"));
+        TeleportRouterLike router = TeleportRouterLike(addr.addr("MCD_ROUTER_TELEPORT_FW_A"));
+        
+        // Sanity checks
+        assertEq(join.line(sourceDomain), line);
+        assertEq(join.fees(sourceDomain), address(fee));
+        assertEq(dai.allowance(escrow, gateway), type(uint256).max);
+        assertEq(dai.allowance(gateway, address(router)), type(uint256).max);
+        assertEq(TeleportFeeLike(fee).fee(), expectedFee);
+        assertEq(TeleportFeeLike(fee).ttl(), expectedTtl);
+        assertEq(router.gateways(sourceDomain), gateway);
+        assertEq(router.domains(gateway), sourceDomain);
+        assertEq(TeleportBridgeLike(gateway).l1Escrow(), escrow);
+        assertEq(TeleportBridgeLike(gateway).l1TeleportRouter(), address(router));
+        assertEq(TeleportBridgeLike(gateway).l1Token(), address(dai));
+
+        {
+            // NOTE: We are calling the router directly because the bridge code is minimal and unique to each domain
+            hevm.startPrank(gateway);
+            router.requestMint(TeleportGUID({
+                sourceDomain: sourceDomain,
+                targetDomain: targetDomain,
+                receiver: bytes32(uint256(uint160(address(this)))),
+                operator: bytes32(0),
+                amount: uint128(toMint),
+                nonce: 0,
+                timestamp: uint48(block.timestamp - TeleportFeeLike(fee).ttl())
+            }), 0, 0);
+            hevm.stopPrank();
+            assertEq(dai.balanceOf(address(this)), toMint);
+            assertEq(join.debt(sourceDomain), int256(toMint));
+        }
+
+        // Check oracle auth mint -- add custom signatures to test
+        uint256 fee = toMint * expectedFee / WAD;
```

Comment:
"This declaration shadows an existing declaration."

Reply:
fixed